beetsplug/zero.py

```diff
@@ -41,6 +41,7 @@ def __init__(self):
 
         self.config.add({
             'fields': [],
+            'update_database': False,
```

Comment:
The default is `False` (at least for now) because changing it would affect users who are relying on the current behavior. IIRC this was the default behavior at one point but it was removed entirely in favour of only updating files' tags.